packages/atomic/src/components/common/generated-answer/styles/generated-answer.pcss

```diff
@@ -66,3 +75,30 @@
     @mixin mobile-footer-xs-layout;
   }
 }
+
+[part='generated-answer-footer'] {
+  &.show-button-hidden {
+    @apply justify-end;
+    [part='answer-show-button'] {
+      @apply hidden;
+    }
+  }
+
+  @container (max-width: 600px) {
+    @mixin mobile-generated-answer-footer-layout;
+  }
+
+  @screen mobile-only {
+    @mixin mobile-generated-answer-footer-layout;
+  }
+}
+
+[part='generated-container'] {
+  &.answer-collapsed {
+    @apply max-h-[250px] overflow-hidden relative content-[''];
```

Comment:
There should be an option on the component to modify the collapsible height.

Reply:
I talked with Audrey, the collapsible height is not an option in the end, it will always be 250px

Reply:
What would you think of making it `16em` tall so it can scale with the font size?